ui/src/app/utils/access-utils.tsx

```diff
@@ -309,3 +305,13 @@ export const eligibleForTier = (profile: Profile, accessTierShortName: string):
   const rtEligiblity = profile.tierEligibilities.find(t => t.accessTierShortName === accessTierShortName)
   return rtEligiblity?.eligible
 };
+
+export const externalSyncModules = (moduleNames: AccessModule[]) => {
```

Comment:
optional nit: this reads like a noun currently, maybe `syncModulesExternal` or `syncModulesExternally`

Reply:
ok